tests/test_uharfbuzz.py

```diff
@@ -786,6 +786,177 @@ def test_list_names_with_user_names(self, mutatorsans):
             "MutatorMathTest-Medium_Narrow_I",
         ]
 
+    def test_reference_table(self, blankfont):
```

Comment:
I don’t think we need this overly detailed test. It should be enough to check for the table length, and may be the binary data. No need to try to decode it.

Reply:
ok